tasks/coverage/snapshots/parser_typescript.snap

```diff
@@ -4792,6 +4766,24 @@ Expect to Parse: tasks/coverage/typescript/tests/cases/conformance/moduleResolut
    ·     ▲
    ╰────
   help: Try insert a semicolon here
+Expect to Parse: tasks/coverage/typescript/tests/cases/conformance/nonjsExtensions/declarationFileForHtmlFileWithinDeclarationFile.ts
+
+  × Function implementation is missing or not immediately following the declaration.
+    ╭─[typescript/tests/cases/conformance/nonjsExtensions/declarationFileForHtmlFileWithinDeclarationFile.ts:11:5]
+ 10 │ export class HTML5Element extends HTMLElement {
+ 11 │     connectedCallback(): void;
+    ·     ─────────────────
+ 12 │ }
+    ╰────
+Expect to Parse: tasks/coverage/typescript/tests/cases/conformance/nonjsExtensions/declarationFileForHtmlImport.ts
+
+  × Function implementation is missing or not immediately following the declaration.
+    ╭─[typescript/tests/cases/conformance/nonjsExtensions/declarationFileForHtmlImport.ts:11:5]
+ 10 │ export class HTML5Element extends HTMLElement {
+ 11 │     connectedCallback(): void;
+    ·     ─────────────────
+ 12 │ }
+    ╰────
 Expect to Parse: tasks/coverage/typescript/tests/cases/conformance/salsa/plainJSRedeclare3.ts
```

Comment:
this is correct.

`@filename` is speficied to be `component.d.html`, but we are checking it as if it's TS

https://github.com/microsoft/typescript/blob/89e928e8b41368bb0888c6d399854c53a2dbf3d4/tests/cases/conformance/nonjsExtensions/declarationFileForHtmlFileWithinDeclarationFile.ts#L3-L16

Reply:
Where's the documentation for considering `.d.html.ts` a ts definition file lol, we can follow up by setting such files as `.d.ts`.

Reply:
Ok, this is stupid, but here is the commit

https://github.com/microsoft/TypeScript/commit/89e928e8b41368bb0888c6d399854c53a2dbf3d4#diff-5f0fce51104cdb1193245a158673eaf24b263dd4ac2bd5f5a26aa57dfa59e0a8R216

on mobile so not sure if it's linked correctly, but look at

src/harness/compilerImpl.ts#L216

Reply:
in the test runner, probably need to split the files at any @ filename and run each of them through the parser to make the tests more accurate